aclk/README.md

```diff
@@ -53,6 +53,7 @@ configuration uses two settings:
 [global]
   enabled = yes
   cloud base url = https://app.netdata.cloud
+  aclk implementation = legacy
```

Comment:
Isn't  the new option in the `[cloud]` section in netdata.conf?

Reply:
(facepalm)
thx, fixed